src/main/clm_varctl.F90

```diff
@@ -230,6 +231,39 @@ module clm_varctl
 
   real(r8), public  :: o3_ppbv = 100._r8
 
+  ! number of wavelength bands used in SNICAR snow albedo calculation, cenlin
+  integer, public :: snicar_numrad_snw = 5 
+
+  ! type of downward solar radiation spectrum for SNICAR snow albedo calculation cenlin 
+  integer, public :: snicar_solarspec = 1  ! 1->mid-latitude winter;2->mid-latitude summer;3->sub-Arctic winter;
+                                           ! 4->sub-Arctic summer;5->Summit,Greenland,summer;6->High Mountain summer;
+
+  ! snow optics type using different refractive index databases in SNICAR, cenlin
+  integer, public :: snicar_snw_optics = 3   ! 1->Warren (1984);2->Warren and Brandt (2008);3->Picard et al (2016)
+
+  ! dust optics type for SNICAR snow albedo calculation, cenlin
+  integer, public :: snicar_dust_optics = 1 ! 1->Saharan dust (Balkanski et al., 2007, central hematite)
+                                            ! 2->San Juan Mountains dust, CO (Skiles et al, 2017)
+                                            ! 3->Greenland dust (Polashenski et al., 2015, central absorptivity)
+  ! option to turn off aerosol effect in snow in SNICAR
+  logical, public :: snicar_use_aerosol = .true. ! if .false., turn off aerosol deposition flux
+
+  ! option for two different SNICAR radiative transfer solver, cenlin
+  integer, public :: snicar_rt_solver = 2   ! 1->Toon et a 1989 2-stream (Flanner et al. 2007) 
+                                            ! 2->Adding-doubling 2-stream (Dang et al.2019)
+
+  ! option for snow grain shape in SNICAR (He et al. 2017 JC), ceniln
+  integer, public :: snicar_snw_shape = 3   ! 1->sphere; 2->spheroid; 3->hexagonal plate; 4->Koch snowflake
+
+  ! option to activate BC-snow internal mixing in SNICAR (He et al. 2017 JC), ceniln
+  logical, public :: snicar_snobc_intmix = .false.   ! false->external mixing for all BC; true->internal mixing for hydrophilic BC
+
+  ! option to activate dust-snow internal mixing in SNICAR (He et al. 2017 JC), ceniln
+  logical, public :: snicar_snodst_intmix = .false.   ! false->external mixing for all dust; true->internal mixing for all dust
+
+  ! option to activate OC in snow in SNICAR, cenlin
+  logical, public :: DO_SNO_OC = .false.    ! control to include organic carbon (OC) in snow
```

Comment:
List of options to review in tomorrow's ctsm software meeting. Main questions:
- Which to keep? (As also raised in [this post](https://github.com/ESCOMP/CTSM/pull/1861/#issuecomment-1507226737))
- How to maintain effectively ([see this post for a suggestion](https://github.com/ESCOMP/CTSM/pull/1861/files#r1267193074))?

Reply:
2023/8/2 Met with @cenlinhe Mark Flanner @wwieder @dlawrenncar. Thanks, everyone!

According to my notes, we agreed the following:
- rt_solver: keep option 2 only
As I understand it, option 1 is the clm50 option BUT parameters have changed in the parameter file causing answers to change from clm50. We have agreed to move forward **without** a backwards compatibility option.
- snw_optics: keep option 3 only
- @cenlinhe please respond:  Which settings combination comes closest to clm50? (Thank you, @cenlinhe)
- @cenlinhe also please confirm: Do we consider the latest settings combination found in this PR, our default settings?
- For other switches I propose these replacements from integer indices to descriptive options and am open to feedback:
```
select case (snicar_solarspec)
case ('mid_latitude_winter')
   <code>
case ('mid_latitude_summer')
   <code>
case ('sub_arctic_winter')
   <code>
case ('sub_arctic_summer')
   <code>
case ('summit_greenland_summer')
   <code>
case ('high_mountain_summer')
   <code>
end select
```
For the next one I chose regional descriptors rather than literature references or other descriptors. Is that best?
```
select case (snicar_dust_optics)
case ('sahara')
   <code>
case ('san_juan_mtns_colorado')
   <code>
case ('greenland')
   <code>
end select
```
```
select case (snicar_snw_shape)
case ('sphere')
   <code>
case ('spheroid')
   <code>
case ('hexagonal_plate')
   <code>
case ('koch_snowflake')
   <code>
end select
```

Reply:
1. The new naming for each option looks good to me.
2. Yes, the latest settings combination in this PR is our default settings.
3. The following setting combination should produce the closest result to clm50 (I exclude the two settings that we decided to only use one best option you mentioned above):
snicar_numrad_snw = 5 
snicar_solarspec = 1 
snicar_dust_optics = 1 
snicar_use_aerosol = .true. 
snicar_snw_shape = 1 
snicar_snobc_intmix = .false.   
snicar_snodst_intmix = .false.
DO_SNO_OC = .false.

Reply:
Thank you!